src/NuGet/BuildNuGets.csx

```diff
@@ -110,6 +116,8 @@ string[] NonRedistPackageNames = {
     "Microsoft.Net.Compilers",
     "Microsoft.Net.Compilers.netcore",
     "Microsoft.Net.CSharp.Interactive.netcore",
+    "Microsoft.NETCore.Csc",
```

Comment:
Nothing -- that was a terrible name that I chose at random to give .NET CLI something to use. I'd prefer we forget that package exists.